content/manuals/billing/cycle.md

```diff
@@ -38,8 +32,12 @@ final charge is $95 ($100-$5).
 {{< tabs >}}
 {{< tab name="Docker subscription" >}}
 
-Follow these steps to switch from a monthly to annual billing
-cycle for your Docker subscription:
+> [!IMPORTANT]
+>
+> Pay by invoice is only available on first-time checkouts and subscription
```

Comment:
Pay By Invoice isn't performed at first-time checkout though, right? At checkout, we'll have a checkbox that will allow them to specify that the later want to pay by invoice at renewal time. This wording can make it sound like we currently support paying by invoice at initial checkout.